js/src/jit/VMFunctionList-inl.h

```diff
@@ -262,7 +262,7 @@ namespace jit {
   _(ThrowRuntimeLexicalError, js::jit::ThrowRuntimeLexicalError)               \
   _(ThrowUninitializedThis, js::ThrowUninitializedThis)                        \
   _(ToBigInt, js::ToBigInt)                                                    \
-  _(ToStringSlow, js::ToStringSlow<CanGC>)
+  _(ToStringSlow, js::ToStringSlowForVM<CanGC>)
```

Comment:
Instead of modifying the string API for this special case, we can, I think, define ToStringSlowForVM here in this file instead?

Reply:
Done.